docs/assets/style.css

```
@@ -0,0 +1,118 @@
+    --code-bg: #1c1f24;
+  }
+}
+
+* { box-sizing: border-box; }
+
+body {
+  margin: 0;
+  background: var(--bg);
+  color: var(--fg);
+  font: 16px/1.65 -apple-system, BlinkMacSystemFont, "Segoe UI", "Noto Sans",
+        "Noto Sans Tamil", "Latha", "Nirmala UI", Roboto, Helvetica, Arial, sans-serif;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Latha" (font-family-name-quotes)

(font-family-name-quotes)